<launch>
  <!-- Launch the OpenNI2 driver -->
  <include file="$(find openni2_launch)/launch/openni2.launch">
     <arg name="depth_registration" value="true" />
     <arg name="publish_tf" value="true" />
  </include>

  <!-- Run the depthimage_to_laserscan node -->
  <node pkg="depthimage_to_laserscan" type="depthimage_to_laserscan" name="depthimage_to_laserscan" output="screen">
    <remap from="image" to="/camera/depth_registered/image_raw" />
    <remap from="camera_info" to="/camera/depth_registered/camera_info" />
    <remap from="scan" to="/scan" />
    <param name="output_frame_id" value="/camera_depth_frame" />
    <param name="scan_time" value="40.0" />
  </node>
 
 <node pkg="tf" type="static_transform_publisher" name="base_to_camera" args = "0.11 0.0 0.2 0.0 0.0 0.0 /base_link /camera_link 100" />

</launch>
